### ISSUE TITLE
Update setup.py metadata to advertise Python 3.5

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8479]
|[<img alt="mindflayer's avatar" src="https://avatars.githubusercontent.com/u/527325?s=50" width="50" height="50">](https://github.com/mindflayer)| @mindflayer reported|
|-|-|
|Trac ID|trac#8479|
|Type|enhancement|
|Created|2016-06-16 10:10:12Z|
|Branch|https://github.com/twisted/twisted/tree/8479-mindflayer-enable-python|

With this ticket I'm proposing to effectively add the Python 3.5 support to Twisted, adding the related metadata to enable the installation of it via PIP.
I've already added a PR on Github to speed up the process (https://github.com/twisted/twisted/pull/180).

<details><summary>Searchable metadata</summary>

```
trac-id__8479 8479
type__enhancement enhancement
reporter__mindflayer mindflayer
priority__normal normal
milestone__None None
branch__8479_mindflayer_enable_python 8479-mindflayer-enable-python
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1466071812119040 1466071812119040
changetime__1466147545774662 1466147545774662
version__None None
owner__mindflayer mindflayer

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban set owner to @mindflayer|
|-|-|

Thanks for your contribution.

I am not an expert in pypi metadata or how setuptools should work.

Can you please update your branch to include a release notes fragment as described here.
http://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles

The text should be as developer friendly as possible and should help developers what new functionalities are available once we make a new release using your patch...or what problem is now fixed.

It would be great if you could also write an automated tests for that future updated will not introduce a regression.

That text should also help me understand what was broken before and I hope that will help me understand how I can test your fix.

Thanks!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub commented|
|-|-|

In changeset 355d735ddc814a4028dddbf5459ad895c78f1516
```
#!CommitTicketReference repository="" revision="355d735ddc814a4028dddbf5459ad895c78f1516"
Create 8479.misc

Adding file documenting my change for ticket #8479.
```

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

See PR for more comments.

So in the end this is not about enabling anything, but just about updating the metadata.

Will merge. Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset 54f95a4eaf582f17bed436d8decdbb27dedaa281
```
#!CommitTicketReference repository="" revision="54f95a4eaf582f17bed436d8decdbb27dedaa281"
Merge 8479-mindflayer-enable-python: Update metadata to advertise Python 3.5.

Author: mindflayer
Reviewer: adiroiban
Fixes: #8479
```
